diffview.js

```diff
@@ -51,7 +51,8 @@ diffview = {
 		var baseTextName = params.baseTextName ? params.baseTextName : "Base Text";
 		var newTextName = params.newTextName ? params.newTextName : "New Text";
 		var contextSize = params.contextSize;
-		var inline = (params.viewType == 0 || params.viewType == 1) ? params.viewType : 0;
+		var inline = (params.viewType == 0 || params.viewType >= 1) ? params.viewType : 0;
```

Comment:
I think you should update params.viewType to use a 1 instead of params.viewType.
Also I think it would look better if it instead did this (depending on whether viewType is used in more places or not(?):

```
var inline = params.viewType ? 1 : 0;
```